Components/Buttons/buttons.css

```diff
@@ -177,7 +177,7 @@
     color:var(--solid-btn-text-color);
     color:var(--secondary-color);
     background-color: black;
-    padding:15px;
+    padding:1.5rem;
     cursor: pointer;
     border:none;
 }
```

Comment:
![image](https://user-images.githubusercontent.com/65125998/152635166-67174192-5dfc-4f3d-ac6a-a9ffc864bbc7.png)
You have used hover properties but see all the hover are using same opacity:0.7
So you can just do 
.btn:hover{
     opacity:0.7
}
And then you can safely remove other classes